soccer/Processor.cpp

```diff
@@ -191,7 +186,7 @@ void Processor::blueTeam(bool value) {
 
     if (_blueTeam != value) {
         _blueTeam = value;
-        if (_radio) _radio->switchTeam(_blueTeam);
+        if (_radio->_radio) _radio->_radio->switchTeam(_blueTeam);
```

Comment:
It might be a good idea to have the RadioNode type wrapper have a `switchTeam` call so that there will be no implementation specific code elsewhere outside the RadioNode class. It makes changing code later on super duper easy since you don't have to just hope nothing is messed with outside the normal control flow